Emby.Server.Implementations/ApplicationHost.cs

```diff
@@ -1080,7 +1080,7 @@ private IEnumerable<string> GetPlugins(string path, bool cleanup = true)
                     }
                     else
                     {
-                        metafile = dir.Split(new[] { Path.DirectorySeparatorChar }, StringSplitOptions.RemoveEmptyEntries)[^1];
+                        metafile = dir.Split(new[] { Path.DirectorySeparatorChar }, StringSplitOptions.RemoveEmptyEntries)[^1]);
```

Comment:
```suggestion
                        metafile = dir.Split(new[] { Path.DirectorySeparatorChar }, StringSplitOptions.RemoveEmptyEntries)[^1];
```
and given that the Split method ihas its closing  parenthesis before the [^1] i guess this is not correct?